guides/v2.3/inventory/reservations.md

```diff
@@ -0,0 +1,171 @@
+
+* **Outstanding reservations**. Magento totals all the initial reservations that have not been compensated. This number will always be negative. If customer A has a reservation for 10 items, and customer B has a reservation 5 for items, then outstanding reservations for the product total -15. 
+
+Therefore, the merchant can fulfill an incoming order as long as the customer orders less than 40 (55 + -15) units. 
+
+{: .bs-callout .bs-callout-info }
+Backorders and Minimum Quantity Threshold settings also affect the calculation of salable quantities, but they are outside the scope of this topic. For more information about these settings, see [Configuring Inventory Management](https://docs.magento.com/m2/ce/user_guide/catalog/inventory-configure-inventory-management.html) in the _Magento User Guide_.
+
+## Reservation objects
+
+A reservation contains the following information:
+
+Parameter | Data type | Description
+--- | --- | ---
+`reservation_id` | Integer | A system-generated ID
+`stock_id` | Integer | The ID of the stock the product is assigned to
+`sku` | String | The SKU of the product
+`quantity` | Float | The number of items in this reservation
+`metadata` | String | The event type, object type, and object ID for this reservation. For example, `{"event_type":"order_placed","object_type":"order","object_id":"8"} `
+{:style="table-layout:auto;"}
+
+The metadata `event_type` can have the following values:
+
+* `order_placed`
+* `order_canceled`
+* `shipment_created`
+* `creditmemo_created`
+* `invoice_created`
+
+Currently, the metadata object type must be `order`, and the object ID is the order ID. In the future, it might be possible to create a reservation when a customer adds an item to a shopping cart. Each item could be reserved for a fixed amount of time, such as 15 minutes, allowing the customer to reserve items while continuing to shop. When this type of reservation is enabled, the metadata could contain additional types of information.
+
+## Reservation lifecycle
+
+The following example shows the sequence of reservations generated for a simple order.
+
+1. The customer makes a purchase order for 25 units of product `SKU-1`. The reservation contains the following information:
+
+   ```
+   reservation_id = 1
+   stock_id = 1
+   sku = SKU-1
+   quantity = -25
+   event_type = order_placed
+```
+
+2. The customer sends an invoice for 20 items, essentially canceling 5 of the units ordered.
+
+   ```
+   reservation_id = 2
+   stock_id = 1
+   sku = SKU-1
+   quantity = 5
+   event_type = order_canceled
+   ```
+
+3. The merchant ships the remaining 20 units.
```

Comment:
Instead of remaining, suggesting something like purchased? They paid for those 20 via the invoice. Are those remaining?

```suggestion
3. The merchant ships the purchased 20 units.
```